src/bo4e/__init__.py

```diff
@@ -184,6 +183,16 @@
     "Zaehlertyp",
     "ZaehlertypSpezifikation",
     "ZusatzAttribut",
+    "Lokationszuordnung",
```

Comment:
Franzi ist aufgefallen, dass die Objekte hier nach BOs, COMs und ENUMs sortiert sind, magst du die neuen auch in die Reihenfolge mit einreihen?

Reply:
Kannst du auch unten bei den Importen spicken, wenn ich das richtig sehe ^^